backend/src/main/java/net/es/oscars/sb/nso/rest/LiveStatusSdpResult.java

```diff
@@ -0,0 +1,32 @@
+package net.es.oscars.sb.nso.rest;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(callSuper=false)
+public class LiveStatusSdpResult extends LiveStatusResult {
+
```

Comment:
Please add a `vcId` field (JsonProperty 'vc-id") here and populate it with the first part of the `vcId:sdpId` tuple. 

i.e, if the SdpId in the  live-status data is `7002:7005`, it would be mapped to a LiveStatusSdpResult with
vcId: 7002
sdpId:7005

Reply:
my bad! done!